cmd/prow-job-dispatcher/main.go

```diff
@@ -718,32 +719,32 @@ func runAsDaemon(o options, promVolumes *prometheusVolumes) {
 	}
 	c.Start()
 
-	// instead using fsnotify watcher falling back to periodic checks as
```

Comment:
unfortunately with git-sync and depth 1 master info is lost and fallback to new checkout time which is inconvenient